### PR TITLE
Log4j v3 enhancement

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
- Added an option to automatically execute commands using Cortex XDR on all Linux OS connected endpoints.
- Added a link to Apache's official release site for both patched versions (2.15.0-rc2 & 2.16.0).

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 1]
```diff
@@ -1118,37 +1118,59 @@ tasks:
...
             iscontext: true
+      - - operator: isNotEqualString
+          left:
+            value:
+              complex:
+                root: inputs.XDREndpointIDs
+            iscontext: true
+          right:
+            value:
+              simple: ALL
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:1168]: Should add here `ignorecase: true`

Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:1168]: Fixed.

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 2]
```diff
@@ -2499,36 +2638,80 @@ tasks:
...
     task:
-      id: 44e0b4cc-b79d-4a42-8f59-30796d55810e
+      id: 511d52a4-f1f3-4164-8cb7-fdbea90dd7ef
       version: -1
-      name: Check if Cortex XDR - IR is Enabled
-      description: Returns 'yes' if integration brand is available. Otherwise returns
-        'no'
-      scriptName: IsIntegrationAvailable
-      type: condition
+      name: Cortex XDR - Execute commands
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:2648]: Need to change the name of the task of the subplaybook

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 3]
```diff
@@ -2410,12 +2513,12 @@ tasks:
...
-    taskid: 3ecd0399-c442-4d24-8744-75e4fc0696c1
+  "71":
+    id: "71"
+    taskid: 30d08296-84a7-48fa-8dde-271c04f1f9a2
     type: playbook
     task:
-      id: 3ecd0399-c442-4d24-8744-75e4fc0696c1
+      id: 30d08296-84a7-48fa-8dde-271c04f1f9a2
       version: -1
       name: Cortex XDR - Execute commands
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:2523]: Need to change the name of the subplaybook task

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml [hunk 4]
```diff
@@ -2192,10 +2216,10 @@ tasks:
...
   "60":
     id: "60"
-    taskid: 63813eaf-9b18-4965-8da5-0f928389863f
+    taskid: 8ea51d16-1d64-40e6-8d3a-d5a459ffaf70
     type: playbook
     task:
-      id: 63813eaf-9b18-4965-8da5-0f928389863f
+      id: 8ea51d16-1d64-40e6-8d3a-d5a459ffaf70
       version: -1
       name: QRadarFullSearch
```
Comment (altmannyarden) [inline Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:2224]: Also here , for some reason the subplaybook tasks returns to the name of the playbooks and not the name you gave to the task

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml (hunk 2 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:2648]: fixed

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml (hunk 3 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:2523]: fixed

File: Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml (hunk 4 above)
Comment (melamedbn) [reply Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml:2224]: fixed

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**